packages/global/components/marko.json

```diff
@@ -12,6 +12,7 @@
     "template": "./sticky-footer.marko"
   },
   "<global-contextual-header>": {
-    "template": "./site-header/contextual.marko"
+    "template": "./site-header/contextual.marko",
+    "@page-type": "string"
```

Comment:
Remove this or add any possible inputs values that could be passed like block-name that the global-contextual-block is using off of input.  It would error now if you tried to pass it, even though the template has `blockName = input.blockName;` on line 7